tools/create_pr_or_update_existing_one.py

```diff
@@ -159,6 +159,20 @@ def update_own_pr(pr_number: int, access_token: str, base_branch: str, repo):
     response = requests.post(url, headers=headers, json=payload)
     response.raise_for_status()
 
+    url_labels = f'{BASE_URL}/repos/{repo}/issues/{pr_number}/labels'
+    response = requests.get(url_labels, headers=headers)
+    response.raise_for_status()
+
+    remove_label_url = (
+        f'{BASE_URL}/repos/{repo}/issues/{pr_number}/labels/ready%20to%20merge'
+    )
+
+    for label in response.json():
+        if label['name'] == 'ready to merge':
+            response = requests.get(remove_label_url, headers=headers)
+            response.raise_for_status()
+            break
```

Comment:
Do I read this correctly that doing a `GET` on a specific label toggles it?

At any rate, it would be good to add some comments to this block: (1) "The following lines remove the ready-to-merge label if there is a previous PR from this bot that has been closed" (2) "We check whether the label is present in the closed PR, and remove it with a GET if it is."